libc/stdlib/itoa.c

```diff
@@ -0,0 +1,34 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+
+char get_ascii(int number){
+	if(number < 10){
+		return number+48;
+	}else{
+		return number+55;
+	}
+}
+
+void split_to_numbers(unsigned int value, size_t * num_of_char, char* str, int base){
```

Comment:
Can we stick to one convention of pointers (where do we put the *) 
```suggestion
void split_to_numbers(unsigned int value, size_t* num_of_char, char* str, int base){
```